recipe/meta.yaml

```diff
@@ -67,6 +67,7 @@ outputs:
 
   - name: gap-defaults
     script: install-defaults.sh
+    # ... and why does this "output" not need any requirements, when the one before and the one after apparently do need them?
```

Comment:
The requirements here are only to run the `install-defaults.sh` script. The `build.sh` has already happened at this point.